README.md

```diff
@@ -2,115 +2,142 @@
 
-If you don't need to install several versions of LBAF, you can install LBAF as a package in editable mode from the project directory.
+## Installation
 
-From the project directory, run:
- ```shell
-pip install -e .
-```
+LBAF can be installed in two ways:
 
-This will automatically install the required dependencies.
+<details>
+<summary><b>1. Install the LBAF Package <i>(recommended)</i></b></summary>
 
-*Note: pip package manager does not support hosting different versions of the same package in a single python environment*.
+<br />
 
-*Note: Although not required, it's common to locally install the project in "editable" or "develop" mode while you're working on it. This allows your project to be both installed and editable in project form.*
+Users can easily install the latest release of LBAF with:
 
-### Install dependencies
+```shell
+pip install lbaf
+```
 
-If LBAF has not been installed as a package (for example because you have multiple versions of LBAF on your machine), you will run LBAF differently.
+Developers should clone the repo and install the package in editable mode:
 
-Install the dependencies by running the following command:
 ```shell
-cd <project-path>
-pip install -r requirements.txt
+git clone git@github.com:DARMA-tasking/LB-analysis-framework.git
+pip install -e LB-analysis-framework
 ```
 
-In general, it is often recommended during development to use python virtual environments and it is possible to have also environments dedicated to some development branches. For example a Python3.8 environment for the branch 125 could be named "venv38-branch-125" and then you could install lbaf as a package in editable mode inside this environment only.
+</details>
+<details>
+<summary><b>2. Install Dependencies</b></summary>
 
-#### Requirements
+<br />
 
-Requirements are divided into `LBAF dependencies` and `LBAF testing`.
+If you do not wish to install LBAF as a package, simply clone the repo and install dependencies:
 
-`LBAF dependencies` are needed in order to LBAF to work.
+```shell
+git clone git@github.com:DARMA-tasking/LB-analysis-framework.git
+pip install -r LB-analysis-framework/requirements.txt
+```
 
-`LBAF testing` are needed for testing purposes.
+</details>
 
 ## Testing
 
-In order to run tests locally, one needs to install test dependencies:
+Begin by installing the test dependencies in `requirements.txt`.
+
+```shell
+pip install tox coverage pylint pytest anybadge
+```
+
+Then, to run all tests locally:
+
 ```shell
 cd <project-path>
 tox
 ```
 
 The `tox` command will:
-- run all test defined in `tox.ini`
+- run all tests defined in `tox.ini`
 - create the `artifacts` directory in main project path
 - create an html coverage report and a pylint report within the `artifacts` directory
 
 ## Usage
 
-### LBAF
-
-If the lbaf package is installed you can run LBAF using the following command:
+If the `lbaf` package is installed, you can run LBAF using the following command:
```

Comment:
Consistency in usage of the passive voice:
`If the lbaf package is installed, LBAF can be run using the following command:`

Reply:
Fixed